src/manufactured_solns.jl

```diff
@@ -432,11 +517,28 @@ using IfElse
             Si += - num_diss_params.vpa_dissipation_coefficient*Dvpa(Dvpa(dfni))
         end
         if num_diss_params.r_dissipation_coefficient > 0.0 && include_num_diss_in_MMS
-            Si += - num_diss_params.r_dissipation_coefficient*Dr(Dr(dfni))
+            Si += - rfac*num_diss_params.r_dissipation_coefficient*Dr(Dr(dfni))
         end
         if num_diss_params.z_dissipation_coefficient > 0.0 && include_num_diss_in_MMS
             Si += - num_diss_params.z_dissipation_coefficient*Dz(Dz(dfni))
         end
+        nu_krook = collisions.krook_collision_frequency_prefactor
+        if nu_krook > 0.0
+            tempi = vthi^2
+            if collisions.krook_collisions_option == "manual"
+                nuii_krook = nu_krook
+            else # default option
+                nuii_krook = nu_krook * densi * tempi^(-1.5)

```

Comment:
Change the variable name to make slightly clearer that this is not what would be 'normalised temperature' in the current conventions of the code?
```suggestion
            Ti_over_Tref = vthi^2
            if collisions.krook_collisions_option == "manual"
                nuii_krook = nu_krook
            else # default option
                nuii_krook = nu_krook * densi * Ti_over_Tref^(-1.5)
```